openquake/commands/tests/data/expected_converted_nrml.xml

```diff
@@ -0,0 +1,176 @@
+<?xml version="1.0" encoding="utf-8"?>
+<nrml
+xmlns="http://openquake.org/xmlns/nrml/0.5"
+xmlns:gml="http://www.opengis.net/gml"
+>
+    <sourceModel>
+        <pointSource
+        id="2"
+        name="point"
+        tectonicRegion="Stable Continental Crust"
+        >
+            <pointGeometry>
+                <Point>
+                    <pos>
+                        -122.0 38.0
+                    </pos>
+                </Point>
+                <upperSeismoDepth>
+                    0.0
+                </upperSeismoDepth>
+                <lowerSeismoDepth>
+                    10.0
+                </lowerSeismoDepth>
+            </pointGeometry>
+            <magScaleRel>
+                WC1994
+            </magScaleRel>
+            <ruptAspectRatio>
+                0.5
+            </ruptAspectRatio>
+            <truncGutenbergRichterMFD aValue="-3.5" bValue="1.0" maxMag="6.5" minMag="5.0"/>
+            <nodalPlaneDist>
+                <nodalPlane dip="90.0" probability="0.3" rake="0.0" strike="0.0"/>
+                <nodalPlane dip="45.0" probability="0.7" rake="90.0" strike="90.0"/>
+            </nodalPlaneDist>
+            <hypoDepthDist>
+                <hypoDepth hypodepth="4.0" probability="0.5"/>
+                <hypoDepth hypodepth="8.0" probability="0.5"/>
+            </hypoDepthDist>
+            <hypoList />
```

Comment:
@ptormene, you should change this. If the hypoList/slipList are missing these nodes must not appear in the XML.